libs/sdk-ui-dashboard/src/presentation/dragAndDrop/draggableWidget/useMoveWidgetHandler.ts

```diff
@@ -0,0 +1,28 @@
+// (C) 2022 GoodData Corporation
+import { useCallback } from "react";
+import { v4 as uuid } from "uuid";
+
+import { useDashboardDispatch } from "../../../model";
+import { moveSectionItemAndRemoveOriginalSectionIfEmpty } from "../../../model/";
+import { InsightDraggableItem, KpiDraggableItem } from "../types";
+
+export function useMoveWidgetDropHandler(sectionIndex: number, itemIndex: number) {
+    const dispatch = useDashboardDispatch();
+
+    const corelationId = "move-widget-" + uuid();
```

Comment:
This will change on every render unconditionally, thus the useCallback below will always revalidate and has no effect. Should the correlationId depend on something?